phpunit.xml.dist

```diff
@@ -10,7 +10,7 @@
 	>
 	<testsuites>
 		<testsuite name="WooCommerce Services Test Suite">
-			<directory suffix=".php">./tests/</directory>
+			<directory prefix="test-" suffix=".php">./tests/php</directory>
```

Comment:
PHPUnit's XML config default on scanning files with `Test.php` suffix. suffix and prefix for our WP/WC file name rules are needed.